engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -4538,7 +4542,7 @@ static void LogFrameBufferError(GLenum status)
                 if (texture_unit == unit)
                 {
                     *index = i;
-                    *type = GetGraphicsType(context->m_CurrentProgram->m_BaseProgram.m_Uniforms[i].m_Type);
+                    *type = context->m_CurrentProgram->m_BaseProgram.m_Uniforms[i].m_Type;
```

Comment:
The "GetGraphicsType" didn't complain that the type had changed :/ But it's no longer needed.